zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/SystemContext.java

```diff
@@ -110,12 +113,27 @@ private void validateConfiguration() {
 
     validateExperimentalConfigs(cluster, brokerCfg.getExperimental());
 
+    validateExporters(brokerCfg.getExporters());
+
     final var security = brokerCfg.getNetwork().getSecurity();
     if (security.isEnabled()) {
       validateNetworkSecurityConfig(security);
     }
   }
 
+  private void validateExporters(final Map<String, ExporterCfg> exporters) {
+    final Set<Entry<String, ExporterCfg>> entries = exporters.entrySet();
+    final Stream<Entry<String, ExporterCfg>> badExporters =
+        entries.stream().filter(entry -> entry.getValue().getClassName() == null);
+    final var badExportersNames = badExporters.map(Entry::getKey).sorted().toList();
```

Comment:
:wrench: can be combined.
```suggestion
     final var badExportersNames = exporters.entrySet().stream().filter(entry -> entry.getValue().getClassName() == null).map(Entry::getKey).sorted().toList();
```

Also, as I discussed before, there is no need to sort here.

Reply:
Thanks. I've applied your suggestion and also removed the sorts.